Lib/fontTools/misc/bezierTools.py

```diff
@@ -831,9 +786,8 @@ def quadraticPointAtT(pt1, pt2, pt3, t):
     Returns:
         A 2D tuple with the coordinates of the point.
     """
-    x = (1 - t) * (1 - t) * pt1[0] + 2 * (1 - t) * t * pt2[0] + t * t * pt3[0]
-    y = (1 - t) * (1 - t) * pt1[1] + 2 * (1 - t) * t * pt2[1] + t * t * pt3[1]
-    return (x, y)
+    a, b, c = calcQuadraticParameters(pt1, pt2, pt3)
+    return a * (t ** 2) + b * t + c
```

Comment:
Tbh. this was a rather arbitrary decision, and I'll gladly put back the Bernstein form.

The `calc*Parameters()` functions already cast to Vector, so it's either use that, or add a cast here before the Bernstein form.
```python
    pt1, pt2, pt3 = (Vector(pt) for pt in (pt1, pt2, pt3))
```

I'm personally more familiar with the parametrized form than the Bernstein form, so that nudged this decision into that direction, too. But again, just let me know what you prefer and I'll adapt.

Reply:
The Bernstein form is probably faster, so I'm starting to lean towards that form now...

Reply:
> Bernstein form is probably faster

+1